webapp/security/views.py

```diff
@@ -333,5 +342,167 @@ def cve(cve_id):
 # ===
 
 
-def api_create_cve():
-    pass
+def create_cve(cve_id):
```

Comment:
Could we secure these with `authentication_required` please? As we've done with `create_notice` in master.

Reply:
Can we do this after we rebase with master? because I don't want to be copying more code from master

Reply:
That feels pretty risky. It would be easy for this to accidentally go live with no auth.

Reply:
Ok, so after I implemented this, I cannot QA until we can generate a discharge in the script?

Reply:
Done. Although the script is not working now due to lack of macaroon discharge. I am working on this see if I can come up with one

Reply:
I can help fix up the script

Reply:
Yeah, but I think it's working for the purposes of this API now. I am expecting that the sec guys will introduce some changes, so once this API is tested, I can focus on search and the script